app/views/payment_gateways/_stripe_elements.html.erb

```diff
@@ -0,0 +1,146 @@
+<script type="text/javascript" src="https://js.stripe.com/v3/"></script>
+
+<div class="well">
+
+  <a id="toggle-stripe-form" class="<%= current_account.credit_card_stored? ? 'card-on-file' : 'hidden' %> pull-right" href="#">
+    <i class="fa fa-pencil"></i>
+    <span>Edit Credit Card Details</span>
+  </a>
+  <br/>
+
+  <%= semantic_form_for 'stripe',
+                        url: hosted_success_admin_account_stripe_path,
+                        method: 'POST',
+                        html: { id: 'stripe-form',
+                                class: "#{current_account.credit_card_stored? ? 'hidden' : 'card-missing'} form-horizontal",
+                                data: {stripe_publishable_key: @stripe_publishable_key}
+                        } do |form| %>
+
+    <fieldset>
+      <legend>Credit card details </legend>
+      <div id="card-element">
+        <!-- a Stripe Element will be inserted here. -->
+      </div>
+
+      <!-- Used to display form errors -->
+      <div id="card-errors" role="alert"></div>
+
+      <%= render partial: 'payment_gateways/stripe_billing_address', object: current_account.billing_address %>
+    </fieldset>
+
+    <fieldset>
+      <div class="form-group">
+        <div class="col-md-10 operations">
+          <%= form.submit 'Save details', class: "btn btn-primary pull-right"%>
+        </div>
+      </div>
+    </fieldset>
+  <% end %>
+</div>
+
+<script type="text/javascript" charset="utf-8">
+    var stripe = Stripe('<%= @stripe_publishable_key %>');
+    var elements = stripe.elements();
+
+    var card = elements.create('card', {
+        iconStyle: 'solid',
+        style: {
+            base: {
+                iconColor: '#8898AA',
+                color: 'black',
+                fontWeight: 300,
+                fontFamily: '"Helvetica Neue", Helvetica, sans-serif',
+                fontSize: '19px',
+                '::placeholder': {
+                    color: '#3f454c',
+                },
+            },
+            invalid: {
+                iconColor: '#e85746',
+                color: '#e85746',
+            }
+        },
+        classes: {
+            focus: 'is-focused',
+            empty: 'is-empty',
+        },
+    });
+    card.mount('#card-element');
+
+    var inputs = document.querySelectorAll('input.field');
+    Array.prototype.forEach.call(inputs, function(input) {
+        input.addEventListener('focus', function() {
+            input.classList.add('is-focused');
+        });
+        input.addEventListener('blur', function() {
+            input.classList.remove('is-focused');
+        });
+        input.addEventListener('keyup', function() {
+            if (input.value.length === 0) {
```

Comment:
I propose using [toggle](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods) instead 🤔 Like this:
```javascript
input.addEventListener('keyup', function() {
    input.classList.toggle('is-empty')
}
```